config_example.yaml

```diff
@@ -7,16 +7,21 @@ chihaya:
     announce: 10m
     min_announce: 5m
     announce_middleware:
-      # These are currently fake values
-      - name: prometheus
-      - name: store_client_validation
-        config:
-          be_clever: true
-      - name: store_create_on_announce
+#      - name: ip_blacklist
+#      - name: ip_whitelist
+#      - name: client_blacklist
+#      - name: client_whitelist
+#      - name: infohash_blacklist
+#      - name: infohash_whitelist
+#      - name: varinterval
+#      - name: deniability
+      - name: store_swarm_interaction
```

Comment:
This doesn't exist yet.

I also poked around with implementing this a bit this weekend and found we need to add some methods to our peer store.